config/v1/types_cluster_version.go

```diff
@@ -215,6 +222,71 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionCapability enumerates optional, core cluster components.
+type ClusterVersionCapability string
+
+const (
+	// ClusterVersionCapabilityOpenShiftSamples manages the sample
+	// image streams and templates stored in the openshift
+	// namespace, and any registry credentials, stored as a secret,
+	// needed for the image streams to import the images they
+	// reference.
+	ClusterVersionCapabilityOpenShiftSamples ClusterVersionCapability = "openshift-samples"
```

Comment:
don't pieces of this get used to identify the correct must-gather images?

@soltysh we should get an e2e created for must-gather to be sure the matching must-gather image is used instead of the backup.

Reply:
pieces yes, not pieces that will get disabled by this, however.
this file will not be part of the samples operator cap: https://github.com/openshift/cluster-samples-operator/blob/master/manifests/08-openshift-imagestreams.yaml

Reply:
(I would argue those imagestreams are not part of the "samples operator" component at all, they just got shoved into that repo out of convenience.  So disabling the samples operator capability will not disable those imagestreams)